app/views/conferences/_conference_details.html.haml

```diff
@@ -41,7 +41,8 @@
             - if !current_user.nil? && current_user.proposal_count(conference) > 0
               = link_to "My Proposals", conference_program_proposals_path(conference.short_title), class: 'btn btn-default'
             - elsif can? :new, conference.program.events.new
-              = link_to "Submit Proposal", new_conference_program_proposal_path(conference.short_title), class: 'btn btn-default'
+              - if !conference.program.cfp.nil?
```

Comment:
Please avoid double negation and use `unless`.

Reply:
I need to enable Rubocop for those things... :muscle:

Reply:
@hennevogel thank you, will update it. :smile: